plugins/obs-text/gdiplus/obs-text.cpp

```diff
@@ -737,6 +743,9 @@ inline void TextSource::Update(obs_data_t *s)
 		if (!text.empty())
 			text.push_back('\n');
 	}
+	if(all_caps)
+		transform(text.begin(), text.end(),
```

Comment:
Style:  This line does not _need_ to have a line-break.
```suggestion
		transform(text.begin(), text.end(), text.begin(), towupper);
```